### PR TITLE
1password cli

### PR DESCRIPTION
**Pull request by DanielHabenicht**
- [x] Have you signed the [Contributor License Agreement](https://cla.opensource.microsoft.com/microsoft/winget-pkgs)?
- [x] Have you checked that there aren't other open [pull requests](https://github.com/microsoft/winget-pkgs/pulls) for the same manifest update/change?
- [x] Have you [validated](https://github.com/microsoft/winget-pkgs/blob/master/AUTHORING_MANIFESTS.md#validation) your manifest locally with `winget validate --manifest <path>`?
- [x] Have you tested your manifest locally with `winget install --manifest <path>`?
- [ ] Does your manifest conform to the [1.2 schema](https://github.com/microsoft/winget-pkgs/tree/master/doc/manifest/schema/1.2.0)?

Note: `<path>` is the name of the directory containing the manifest you're submitting.

-----

###### Microsoft Reviewers: [Open in CodeFlow](https://portal.fabricbot.ms/api/codeflow?pullrequest=https://github.com/microsoft/winget-pkgs/pull/92371)

### CODE REVIEW

File: manifests/a/AgileBits/1PasswordCLI/2.12.0/AgileBits.1PasswordCLI.installer.yaml [hunk 1]
```diff
@@ -0,0 +1,27 @@
+# Created with YamlCreate.ps1 v2.2.1 $debug=NVS0.CRLF.5-1-22621-963.Win32NT
+# yaml-language-server: $schema=https://aka.ms/winget-manifest.installer.1.2.0.schema.json
+
+PackageIdentifier: AgileBits.1PasswordCLI
```
Comment (Trenly) [inline manifests/a/AgileBits/1PasswordCLI/2.12.0/AgileBits.1PasswordCLI.installer.yaml:4]:
```suggestion
PackageIdentifier: AgileBits.1Password.CLI
```
This should probably be a subpackage of AgileBits.1Password

File: manifests/a/AgileBits/1PasswordCLI/2.12.0/AgileBits.1PasswordCLI.locale.en-US.yaml [hunk 2]
```diff
@@ -0,0 +1,30 @@
+# Created with YamlCreate.ps1 v2.2.1 $debug=NVS0.CRLF.5-1-22621-963.Win32NT
+# yaml-language-server: $schema=https://aka.ms/winget-manifest.defaultLocale.1.2.0.schema.json
+
+PackageIdentifier: AgileBits.1PasswordCLI
```
Comment (Trenly) [inline manifests/a/AgileBits/1PasswordCLI/2.12.0/AgileBits.1PasswordCLI.locale.en-US.yaml:4]:
```suggestion
PackageIdentifier: AgileBits.1Password.CLI
```
This should probably be a subpackage of AgileBits.1Password

File: manifests/a/AgileBits/1PasswordCLI/2.12.0/AgileBits.1PasswordCLI.yaml [hunk 3]
```diff
@@ -0,0 +1,8 @@
+# Created with YamlCreate.ps1 v2.2.1 $debug=NVS0.CRLF.5-1-22621-963.Win32NT
+# yaml-language-server: $schema=https://aka.ms/winget-manifest.version.1.2.0.schema.json
+
+PackageIdentifier: AgileBits.1PasswordCLI
```
Comment (Trenly) [inline manifests/a/AgileBits/1PasswordCLI/2.12.0/AgileBits.1PasswordCLI.yaml:4]:
```suggestion
PackageIdentifier: AgileBits.1Password.CLI
```
This should probably be a subpackage of AgileBits.1Password

**Review by Trenly — CHANGES_REQUESTED** [review]

**Comment by Trenly** [comment]
@msftbot zip

**Review by ItzLevvie — DISMISSED** [review]

**Comment by Trenly** [comment]
@msftbot reset feedback

**Review by Trenly — CHANGES_REQUESTED** [review]
https://github.com/microsoft/winget-pkgs/pull/92371#discussion_r1057790701

**Review by Trenly — APPROVED** [review]

**Pull request merged**